packages/common/src/users/_internal/UserUiSchemaSettings.ts

```diff
@@ -0,0 +1,45 @@
+import { IArcGISContext } from "../../ArcGISContext";
+import { IUiSchema, UiSchemaRuleEffects } from "../../core";
+import { IHubUser } from "../../core/types/IHubUser";
+
+/**
+ * @private
+ * constructs the settings uiSchema for Hub Users.
+ * This defines how the schema should be rendered in the
+ * user workspace settings pane
+ * @param i18nScope - translation scope to be interpolated into the uiSchema
+ * @param options - additional options to customize the uiSchema
+ * @param context - contextual auth and portal information
+ * @returns
+ */
+export const buildUiSchema = async (
+  i18nScope: string,
+  options: any,
+  context: IArcGISContext
+): Promise<IUiSchema> => {
+  return {
+    type: "Layout",
+    elements: [
+      {
+        type: "Section",
+        labelKey: `${i18nScope}.sections.userSettings`,
+        elements: [
+          {
```

Comment:
Not entirely sure _where_ the best place to put a note/comment would be (maybe in the harness caveats section?):
<img width="450" alt="Screenshot 2024-08-30 at 11 16 16 AM" src="https://github.com/user-attachments/assets/23ea04e9-4811-4183-960e-50bca8f1a6be">

But I went to verify functionality of this toggle in the harness, and I don't _think_ it works there because the user hub settings don't appear to be populated in the harness? Not entirely sure what's going on (maybe it's just me), but everything appears to work correctly on QA.